src/server/actions/webhooks.ts

```diff
@@ -0,0 +1,33 @@
+import dbConnect from '@/utils/db-connect';
+import CMError, { CMErrorType } from '@/utils/cmerror';
+import { BackgroundCheckWebhookPayload } from '@/types/dataModel/backgroundCheckWebhookPayload';
+import VolunteerSchema from '@/server/models/Volunteer';
+/**
+ * Get a specific volunteer and update background check
+ * @param payload // BackgroundCheckWebhookPayload
+ * @returns // volunteerschema or null
+ */
+export async function handleBackgroundCheckWebhook(
+  payload: BackgroundCheckWebhookPayload
+): Promise<void> {
+  let res;
+  try {
+    await dbConnect();
+    res = await VolunteerSchema.findOneAndUpdate(
```

Comment:
if res == null return 404

Reply:
why?

Reply:
oh I guess you meant to say return a 404 in the API if res == null